simulation_parameters/microbe_stage/membranes.json

```diff
@@ -97,14 +97,14 @@
     "AlbedoTexture": "res://assets/textures/SilicaMembrane.png",
     "NormalTexture": "res://assets/textures/SilicaMembrane_normal.png",
     "DamagedTexture": "res://assets/textures/SilicaMembraneDamaged.png",
-    "MovementFactor": 0.8,
+    "MovementFactor": 0.6,
     "BaseWigglyness": 0.25,
     "MovementWigglyness": 0.45,
     "OsmoregulationFactor": 0.6,
     "ResourceAbsorptionFactor": 0.6,
-    "Hitpoints": 140,
+    "Hitpoints": 180,
     "PhysicalResistance": 1.6,
-    "ToxinResistance": 0.6,
```

Comment:
Doesn't this mean that there is no longer a counter to the silica membrane? (as it is both resistant to physical and toxin damage)